vis.py

```diff
@@ -521,7 +521,7 @@ def tabulate(
 		val_dataset_name = f[f.find('transcripts_') + len('transcripts_'):eidx]
 		checkpoint = os.path.join(experiment_dir, 'checkpoint_' + f[eidx:].replace('.json', '.pt')) if not json_ else f
 		metric = 'wer' if wer else 'entropy' if entropy else 'loss' if loss else 'per' if per else 'der' if der else 'cer'
-		val = torch.tensor([j[metric] for j in json.load(open(f)) if j['labels'].startswith(labels.alphabet)] or [0.0])
+		val = torch.tensor([j[metric] for j in json.load(open(f)) if j['labels_name'].startswith(labels.alphabet)] or [0.0])
```

Comment:
r["labels"] doesn't exist anymore, now it's called r["labels_name"], according to vadimkantorov (metrics.py line 101)